ui/components/ui/disclosure/disclosure.constants.ts

```diff
@@ -0,0 +1,4 @@
+export enum DISCLOSURE_TYPES {
+  DEFAULT = 'default',
+  ARROW = 'arrow',
+}
```

Comment:
I believe our enum conventions are to use PascalCase and singular and possibly best not to use the word types as an enum can also be used as a type

https://www.notion.so/Use-TypeScript-9ec7bd8fc8cf4d82b81cd8943c4707cd?pvs=4#336ec1a7c32f44feb02cc8f14727620d

```suggestion
export enum DisclosureVariant {
  Default = 'default',
  Arrow = 'arrow',
}
```